File: Selenium/Activities/activity16.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity16 {
    public static void main(String[] args) throws InterruptedException {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/selenium/dynamic-attributes");
        System.out.println(driver.getTitle());
        driver.findElement(By.xpath("//input[contains(@class, '-username')]")).sendKeys("admin");
        driver.findElement(By.xpath("//input[contains(@class, '-password')]")).
                sendKeys("password");
        driver.findElement(By.xpath("//label[text()= 'Confirm Password']/following-sibling::input"))
                .sendKeys("password");
        driver.findElement(By.xpath("//label[contains(text(), 'Email')]/following-sibling::input"))
                .sendKeys("1234@gmail.com");
        driver.findElement(By.xpath("//button[contains(text() , 'Sign Up')]")).click();
        String Confirmation_message = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Message: " + Confirmation_message);

    }

}

File: Selenium/Activities/activity16.py
from telnetlib import EC

from selenium import webdriver
from selenium.webdriver import ActionChains, Keys
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support import expected_conditions
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager
service = FirefoxService(GeckoDriverManager().install())



with webdriver.Firefox(service=service) as driver:

    wait = WebDriverWait(driver, 10)


    driver.get("https://www.training-support.net/selenium/dynamic-attributes")
    # Print the title of the page
    print("Page title is: ", driver.title)


    username = driver.find_element(By.XPATH, "//input[contains(@class, '-username')]")

    password = driver.find_element(By.XPATH, "//input[contains(@class, '-password')]")

    confirm_password = driver.find_element(By.XPATH, "//label[contains(text(), 'Confirm Password')]//following::input")


    email = driver.find_element(By.XPATH, "//input[contains(@class, 'email-')]")


    username.send_keys("Deku")
    password.send_keys("PlusUltra!")
    confirm_password.send_keys("PlusUltra!")
    email.send_keys("deku@ua.edu")

    driver.find_element(By.XPATH, "//button[text()='Sign Up']").click()


    wait.until(EC.visibility_of_element_located((By.ID, "action-confirmation")))

    message = driver.find_element(By.ID, "action-confirmation").text
    print("Success message: ", message)